release-team/release-team-removal.md

```diff
@@ -0,0 +1,48 @@
+# Resignation
+Voluntary resignation is outlined in the [shadow handbook](https://github.com/leonardpahlke/sig-release/blob/master/release-team/shadows.md). Role lead should discuss with Release Lead and Emiritus Advisor to find a replacement if they would like to step down.
+
+# Removal Process
+Remaining on the team without contributing creates an unhealthy environment for the rest of the team, and negatively affect the outcome of the release. We strive to be inclusive and empathetic to individuals circumstances. 
+The following outline is not meant to be definitive or punitive but rather outlines general guidance for leads on the process.  
+## Shadow Removal Guideline
+
+Reasons for removal could be including but not limited to:
+- Failure to attend shadow orientation hosted by EA (we host 2-3 sessions in different timezone)
+- Failure to attend shadow orientation hosted by role lead (if lead doesn't offer timezone-appropriate meeting, shadow should communicate to lead about obtaining and watching the recording)
+- Failure to attend 3 consecutive release meeting without communication to role lead
+- Failure to perform shadow tasks in due time, after 2 discussions with role lead about performance
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on the team's responsibility
+- Violation of CNCF Code of Conduct
+
+Note that communications ahead of time to inform the leads or find alternative arrangements are greatly preferred in these circumstances.
+### Process
+It is important to be kind and patient while trying to resolve the situation.
```

Comment:
I would add a bullet point to clarify this point `Failure to perform shadow tasks in due time, after 2 discussions with role lead about performance`. I don't think this would immediately trigger reaching out to the EA. I would first reach out to the shadow directly over slack to check in. Personally, if after 2 discussions of the shadow failing to perform tasks on time, then I would only raise it to the EA. 

```suggestion
### Process
It is important to be kind and patient while trying to resolve the situation.
- Notify the shadow in private message over slack of missed deadline and first warning.
- On second occurrence, schedule 1:1 meeting with shadow over slack and discuss performance.
```